libsolidity/interface/StandardCompiler.cpp

```diff
@@ -1482,11 +1482,17 @@ Json StandardCompiler::compileSolidity(StandardCompiler::InputsAndSettings _inpu
 
 		// IR
 		if (compilationSuccess && isArtifactRequested(_inputsAndSettings.outputSelection, file, name, "ir", wildcardMatchesExperimental))
-			contractData["ir"] = compilerStack.yulIR(contractName);
+		{
+			std::optional<std::string> const& ir = compilerStack.yulIR(contractName);
+			contractData["ir"] = ir ? *ir : std::string{};
```

Comment:
I guess it is more readable like:
```suggestion
			contractData["ir"] = ir.value_or(std::string{});
```

Reply:
fair enough!